addons/mrp/wizard/change_production_qty.py

```diff
@@ -100,4 +100,8 @@ def change_prod_qty(self):
                 moves_finished = production.move_finished_ids.filtered(lambda move: move.operation_id == operation) #TODO: code does nothing, unless maybe by_products?
                 moves_raw.mapped('move_line_ids').write({'workorder_id': wo.id})
                 (moves_finished + moves_raw).write({'workorder_id': wo.id})
+
+        # run scheduler for moves forecasted to not have enough in stock
+        wizard.mo_id.filtered(lambda mo: mo.state in ['confirmed', 'progress']).move_raw_ids._trigger_scheduler()
```

Comment:
`self` instead of `wizard` ? (the method can be batch obviously)

Reply:
oops, it's a mistake, it has to be `self`.